src/utils/identitiesUtils.ts

```diff
@@ -42,12 +42,11 @@ import {
 import {
 	centrifugeAmberMetadata,
 	centrifugeMetadata,
-	defaultMetaData,
 	edgewareMetadata,
 	kulupuMetadata,
 	kusamaMetadata,
 	polkadotMetaData,
-	substrateDevMetadata,
```

Comment:
Is this not supported/useful anymore? Why not?

Reply:
I believe it was just replaced with rococo network instead.

Reply:
This was used back then with a node running in  `- - dev` (the one hosted on dev-node.substrate.dev). It's still live, but barely updated and used. It's reset every hour, but last I checked it was not even running substrate v2, so you can forget about it and safely remove it.